sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -164,9 +168,16 @@ def tune(
         self,
         # TODO (andreyvelich): How to be consistent with other APIs (name) ?
         name: str,
-        objective: Callable,
-        parameters: Dict[str, Any],
-        base_image: str = constants.BASE_IMAGE_TENSORFLOW,
+        model_provider_parameters: Optional[Any] = None,
+		dataset_provider_parameters: Optional[Any] = None,
+		storage_config: Optional[Dict[str, Optional[Union[str, List[str]]]]] = {
+            "size": constants.PVC_DEFAULT_SIZE,
+            "storage_class": None,
+            "access_modes": constants.PVC_DEFAULT_ACCESS_MODES,
+        },
```

Comment:
Please can you format your code with `black` ?

Reply:
Yes, of course. I will do that after I made all the changes.

Reply:
@andreyvelich I have reformatted the code with Black, but I noticed that some comments are too long and do not wrap automatically. Do I need to adjust it too? If so, is there a specific tool I can use for this?

Reply:
Yes, please adjust them to follow the max line width

Reply:
Got it! I have reformatted it with max line length of 79.